chapters/object_types.txt

```diff
@@ -52,6 +52,43 @@ Feature support information can be queried as properties on the `ANARIRenderer`
 | feature | `STRING_LIST` | Yes      | list of supported features
 |===================================================================================================
 
+
+Feature: `KHR_RENDERER_BACKGROUND_COLOR`
+
+.Parameters understood by renderers with feature `KHR_RENDERER_BACKGROUND_COLOR`.
+[cols="<,<,>,<3",options="header,unbreakable"]
+|===================================================================================================
+| Name       | Type         |      Default | Description
+| background |`FLOAT32_VEC4`| (0, 0, 0, 1) | the background color
```

Comment:
Although we have a framebuffer format with alpha channel, we never specified what it means/holds. With the `background` color we now also have alpha &ndash; shouldn't the background be transparent per default ?

Reply:
I think it's good for it to be opaque for 2 reasons: it's surprising when the background is transparent and `vec4(0,0,0,1)` is our default attribute value in other places. I think consistency with attributes is a good tiebreaker here.